developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,403 @@
+
+The [fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move) provides a standard, type-safe framework for defining fungible assets within the Aptos Move ecosystem.
+
+In this standard, fungible assets are stored in `Object<FungibleStore>` that has a specific amount of units. Fungible assets are units that are interchangeable with others of the same metadata. The standard is built upon object model so all the resources defined here are included in object resource group and stored inside objects. The fungible assets within an object can be divided into smaller units creating new stores without creating new units. Similarly they can be combined to aggregate units into fewer objects. The standard also supports minting new units and burning existing units with appropriate controls.
+
+The relationship between the structures laid out in this standard is shown in this diagram.
+<div style={{textAlign: "center"}}>
+<ThemedImage
+alt="fungible asset architecture"
+sources={{
+    light: useBaseUrl('/img/docs/fungible-asset.svg'),
+    dark: useBaseUrl('/img/docs/fungible-asset-dark.svg'),
+  }}
+/>
+</div>
+
+## Difference with Aptos Coin
+
+FA is a broader category than just coins. While fungible coins are just one possible use case of FA, it can represent a wider range of fungible items, such as in-game assets like gems or rocks, event tickets, and partial ownership of real-world assets. FA is constructed using an object model, which provides the flexibility for customizable, detailed management and offers a new programming model based on objects.
+
+Minimally, [Aptos coin](./aptos-coin.md) should be interchangeable with FA, it is up to the will of the ecosystem to dictate whether or not Aptos coin is replaced by a FA equivalent.
+
+## Structures
+
+### Metadata Object
+
+Within FA, metadata defines attributes of the type and other common features. The type itself is defined by the Object or address where this information lies. In other words, two assets with identical metadata but distinct Objects are not the same. The metadata layout is defined as:
+
+```rust
+#[resource_group_member(group = aptos_framework::object::ObjectGroup)]
+struct Metadata has key {
+    supply: Option<Supply>,
+    /// Name of the fungible metadata, i.e., "USDT".
+    name: String,
+    /// Symbol of the fungible metadata, usually a shorter version of the name.
+    /// For example, Singapore Dollar is SGD.
+    symbol: String,
+    /// Number of decimals used for display purposes.
+    /// For example, if `decimals` equals `2`, a balance of `505` coins should
+    /// be displayed to a user as `5.05` (`505 / 10 ** 2`).
+    decimals: u8,
+}
+```
```

Comment:
It might also be clearer to give an example here where 0x123::USDT in the current Coin standard can be referred to as just 0x123 in the Fungible Asset standard where 0x123 is the metadata object address